xdsl/irdl.py

```diff
@@ -726,14 +743,14 @@ def get_op_constructs(
 
 def get_attr_size_option(
     construct: VarIRConstruct
-) -> AttrSizedOperandSegments | AttrSizedResultSegments | None:
+) -> AttrSizedOperandSegments | AttrSizedResultSegments | AttrSizedRegionSegments:
     """Get the AttrSized option for this type."""
     if construct == VarIRConstruct.OPERAND:
         return AttrSizedOperandSegments()
     if construct == VarIRConstruct.RESULT:
         return AttrSizedResultSegments()
     if construct == VarIRConstruct.REGION:
-        return None
+        return AttrSizedRegionSegments()
```

Comment:
It seems that this PR contains changes from another one, can you please set the other branch as the target instead of main?

Reply:
Done!